src/client/src/app/actions/account/addUserRecipe.js

```diff
@@ -0,0 +1,32 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+export const addUserRecipe = createAsyncThunk(
+  'account/addRecipe',
+  async ({ recipe }, { getState, rejectWithValue }) => {
+    try {
+      const token = getState()?.account.token;
+      const headers = {
+        'Content-Type': 'application/json',
+        'authorization': token,
+      };
+      const response = await fetch('/api/user/recipes', {
+        method: 'PUT',
+        headers,
+        body: { recipeId: JSON.stringify(recipe.id) },
```

Comment:
make a variable const body = JSON.stringify({ recipeId: recipe.id }); and then pass it to fetch method